centreon/packages/ui/src/Form/Inputs/Checkbox.tsx

```diff
@@ -0,0 +1,96 @@
+import { ChangeEvent } from 'react';
```

Comment:
Please, add some stories for this

Reply:
I moved the code of this component to another PR, this current PR is only for the Listing component